map/routing_manager.cpp

```diff
@@ -250,6 +250,29 @@ drape_ptr<df::Subroute> CreateDrapeSubroute(vector<RouteSegment> const & segment
     return nullptr;
   }
 
+  // We support vizualization of fake edges only in the head and in the tail of subroute.
+  int i = 0;
+  while (i < segments.size() && !segments[i].GetSegment().IsRealSegment())
+  {
+    subroute->m_headFakeDistance = segments[i].GetDistFromBeginningMerc() - baseDistance;
+    i++;
+  }
+
+  i = static_cast<int>(segments.size()) - 1;
+  auto constexpr kEps = 1e-5;
+  auto constexpr kBias = 1.0; // We need some bias to prevent visual artefacts.
```

Comment:
```
// To prevent visual artefacts, in the case when all tail segments are real
// m_tailFakeDistance must be greater than the length of the route.
auto constexpr kBias = 1.0;
```

Reply:
done